src/containers/DeepLink/LoginRequestIdentity/LoginRequestIdentity.js

```diff
@@ -54,17 +93,72 @@ const LoginRequestIdentity = props => {
       } else {
         setLinkedIds({})
       }
+      setIdsloaded(true);
+
     } catch (e) {
       createAlert('Error Loading Linked VerusIDs', e.message);
     }
 
     setLoading(false)
-  }
+  } 
+
+  useEffect(() => {
+    if(extraParams && extraParams.fqn){
+      const data = {[SEND_MODAL_IDENTITY_TO_LINK_FIELD]: extraParams.fqn};
+      openLinkIdentityModal(CoinDirectory.findCoinObj(system_id, null, true), data);
+    }
+  }, [extraParams])
+
+  useEffect(() => {
+
+    if (idsloaded && req.challenge.subject && req.challenge.subject.length > 0 &&
+      req.challenge.subject.some(item => item.vdxfkey === primitives.ID_ADDRESS_VDXF_KEY.vdxfid)) {
+
+        const providionedId = req.challenge.subject.find(item => item.vdxfkey === primitives.ID_ADDRESS_VDXF_KEY.vdxfid).data;
+        for (const chainId of activeCoinIds) { 
+          if (linkedIds[chainId] && Object.keys(linkedIds[chainId]).includes(providionedId)) {
+            return;
+          }
+        }
+      getIdentity(system_id, providionedId).then((provisionedID) => {
+        getPotentialPrimaryAddresses().then((addresses) => {
+          if (provisionedID.result) {
+            for (const address of provisionedID.result.identity.primaryaddresses) {
+              if (addresses.includes(address)) {
+                dispatch({
+                  type: SET_DEEPLINK_DATA_EXTRAPARAMS,
+                  payload: {
+                    extraParams: { fqn: provisionedID.result.fullyqualifiedname }
+                  },
+                });
+                return;
+              }
+            }
+          }
+        })
+      })
+    }
+  }, [idsloaded])
```

Comment:
It seems that the fqn in extraParams is getting passed in through the processVerusId function so this is maybe no longer necessary? If so I'd prefer it to be deleted, as an automatic way to detect if a login-consent request is an interesting concept, but this does a few things incorrectly. Some of those include ignoring ID_FULLYQUALIFIEDNAME_VDXF_KEY as a subject type, which is a valid alternative to ID_ADDRESS_VDXF_KEY, and not accounting for potentially multiple subjects. If its necessary than we should discuss refactoring this.